src/components/CreateNFT.vue

```diff
@@ -0,0 +1,454 @@
+
+    if (this.hasEnoughXor && !this.isXorSufficientForNextTx({ type: Operation.RegisterAsset })) {
+      this.$emit('showHeader');
+      this.showFee = false;
+      this.$emit('stepChange', Step.Warn);
+      return;
+    }
+
+    this.showFee = true;
+    this.$emit('stepChange', Step.ConfirmNFT);
+  }
+
+  async onConfirm(): Promise<void> {
+    await this.withNotifications(async () => {
+      if (!this.hasEnoughXor) {
+        throw new Error('walletSend.badAmount');
+      }
+      if (this.file) {
+        await this.storeNftImage(this.file);
+      }
+      await this.registerNftAsset();
+      this.navigate({ name: RouteNames.Wallet });
+    });
+  }
+
+  confirmNextTxFailure(): void {
+    this.$emit('showHeader');
+    this.showFee = true;
+    this.$emit('stepChange', Step.ConfirmNFT);
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.image-upload {
+  &__input {
+    display: none;
+  }
+
+  &__btn {
+    background: var(--s-color-base-content-tertiary) !important;
+    margin-top: #{$basic-spacing-medium};
+    padding: 0 !important;
+  }
+
+  &__label {
+    padding: calc(var(--s-size-small) / 2);
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  &__filename {
+    margin-left: 10px;
+    font-size: 12px;
+  }
+}
+
+.info-line-container {
+  text-transform: uppercase;
+  margin-bottom: calc(var(--s-size-small) / 2);
+}
+</style>
+
+<style lang="scss">
+.wallet-settings-create-token {
+  &_desc {
+    color: var(--s-color-base-content-primary);
+    font-size: var(--s-font-size-extra-small);
+    font-weight: 300;
+    line-height: var(--s-line-height-base);
+    padding: var(--s-basic-spacing) #{$basic-spacing-small} #{$basic-spacing-medium};
+  }
+
+  &_action {
+    margin-top: #{$basic-spacing-medium};
+    width: 100%;
+  }
+}
+
+.s-textarea {
+  margin-bottom: #{$basic-spacing-medium};
+}
+
+.el-textarea {
+  height: 54px;
+  // margin-top: 6px;
+
+  &__inner {
+    resize: none !important;
+  }
+}
+
+.preview-image {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  overflow: hidden;
+  border: 2px dashed var(--s-color-base-content-tertiary);
+  border-radius: var(--s-border-radius-small);
+  margin: #{$basic-spacing-medium} 0;
+  height: 200px;
+  position: relative;
+
+  .image {
+    margin: 0 auto;
+  }
+
+  .placeholder {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  &__content {
+    height: 176px;
+    width: 176px;
+    object-fit: cover;
+    border-radius: 12px;
+  }
+
+  &__icon {
+    color: var(--s-color-base-content-tertiary) !important;
+    font-size: var(--s-size-small) !important;
+    margin-bottom: calc(var(--s-size-small) / 2);
+  }
+
+  &__placeholder {
+    letter-spacing: var(--s-letter-spacing-small);
+    color: var(--s-color-base-content-secondary);
+    font-weight: 400 !important;
+    font-size: calc(var(--s-size-small) / 2);
+    text-align: center;
+    padding: 0 50px;
+    color: #2a171f;
```

Comment:
`var(--s-color-base-content-primary)` is used in figma and looks good using dark theme too

![nft-dropbox](https://user-images.githubusercontent.com/53777036/149278069-e99ff2e2-36cb-415d-b29d-fa0ddeec4db9.jpg)

Reply:
fixed